element.cpp

```diff
@@ -203,6 +213,17 @@ QString NamedLocation::toQString()
 	return namedLocationQString;
 }
 
+QString NamedLocation::toQStringForSave()
+{
+	QString saveNamedLocationQString = locationToQString();
+	if(text != nullptr)
```

Comment:
Should this not be just `if(!text)`?

Reply:
Can't be changed, change would crash the program